### ISSUE TITLE
[build] An easy way to set up visual studio

### ISSUE THREAD

**Issue by JacobGood1** [raysan5/raylib#252]
1. run the default installer
2. create a win32 console project from vs
3. in Configuration Properties/Linker/General add C:\raylib\raylib\release\win32
4. in Configuration Properties/Linker/Additional Dependencies add raylib.lib
5. in Configuration Properties/VC++ Directories/Include Directories add C:\raylib\raylib\release\win32
6. Configuration Properties/VC++ Directories/Library Directories add C:\raylib\raylib\release\win32\mingw32
7. copy the raylib.dll from C:\raylib\raylib\release\win32\mingw32 to the vs projects folder

8. open powershell and type cd C:\raylib\raylib\release\win32\mingw32
9. type dumpbin /exports raylib.dll
10. copy all the names ONLY, no numbers
11. create a file in that same folder, C:\raylib\raylib\release\win32\mingw32, called raylib.def
12. go into the file type EXPORTS at the top
13. paste all the names from step 10 into the file and save the file
14. in powershell type lib /def:C:\raylib\raylib\release\win32\mingw32\raylib.def /OUT:C:\raylib\raylib\release\win32\mingw32\raylib.lib

that is it, visual studio can now use raylib, happy hacking!

ps. maintainer, you could create a lib file for them from steps 8 to 14 and they could skip those steps

**Comment by JacobGood1** [comment]
whoops, looks like I got too excited and did not read the docs =)
(I do that all the time)

**Comment by raysan5** [comment]
Hey! No problem! Really appreciate any try to help and contributing! :D

Just let me know if it works ok! ;)

**Comment by Phariax** [comment]
Hi guys, and sorry for my english.


I've downloaded raylib, and I would like to use it with Visual Studio 2015. I am newbie at programming, but I am learning for a while now. So my problem is, when I want to build the projects, I've got a bunch of errors. I suppose that errors occurs, because of two header files are missing, or cant read. This two are : Cannot open include file: 'AL/al.h': No such file or directory, and Cannot open include file: 'GLFW/glfw3.h': No such file or directory. 

I've tried to going through the configure description (raylib/wiki/Create-Visual-Studio-Projectstep ) step by step. Sadly, without any success, the two errors still there. Can anybody help me? 

When I am using notepad++ version, it is fine and cool. (unfortunately it has no stepping mode debugging feature, as far as I know.)

Thanks

Phar
